# Translations template for vigilo-connector-metro.
# Copyright (C) 2011 ORGANIZATION
# This file is distributed under the same license as the vigilo-connector-metro
# project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2011.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: vigilo-connector-metro 2.0.0\n"
"Report-Msgid-Bugs-To: contact@projet-vigilo.org\n"
"POT-Creation-Date: 2011-03-22 11:14+0100\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.4\n"

#: src/twisted/plugins/vigilo_metro.py:41
msgid ""
"Please set the path to the configuration database generated by VigiConf in "
"the settings.ini."
msgstr ""

#: src/vigilo/connector_metro/nodetorrdtool.py:101
#, python-format
msgid "Host '%(host)s' with datasource '%(ds)s' not found in the configuration !"
msgstr ""

#: src/vigilo/connector_metro/nodetorrdtool.py:122
#, python-format
msgid "RRDtool could not create the file: %(filename)s. Message: %(msg)s"
msgstr ""

#: src/vigilo/connector_metro/nodetorrdtool.py:134
#, python-format
msgid "'%(msgtype)s' is not a valid message type for metrology"
msgstr ""

#: src/vigilo/connector_metro/nodetorrdtool.py:144
#, python-format
msgid "Not a valid performance message (missing '%(tag)s' tag)"
msgstr ""

#: src/vigilo/connector_metro/nodetorrdtool.py:152
#, python-format
msgid "Skipping perf update for host %s"
msgstr ""

#: src/vigilo/connector_metro/nodetorrdtool.py:178
#, python-format
msgid "Unable to create the directory '%s'"
msgstr ""

#: src/vigilo/connector_metro/nodetorrdtool.py:254
#, python-format
msgid "RRDtool could not update the file %(filename)s. Message: %(msg)s"
msgstr ""

#: src/vigilo/connector_metro/nodetorrdtool.py:272
msgid "Received signal to reload the configuration"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:99
#, python-format
msgid "Unable to start \"%(rrdtool)s\". Make sure the path is correct."
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:102
#, python-format
msgid ""
"Unable to start \"%(rrdtool)s\". Make sure RRDtool is installed and you have "
"permissions to use it."
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:115
#, python-format
msgid "The RRD directory does not exist: %s"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:124
#, python-format
msgid "Unable to create directory '%(dir)s'"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:128
#, python-format
msgid "Unable to write in the directory '%(dir)s'"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:193
#, python-format
msgid "Started RRDtool subprocess: pid %(pid)d"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:214
msgid "The process has not yet completed the previous job"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:247
msgid "No deferred available in _handle_result(), this should not happen"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:268
msgid "The RRDTool process exited normally"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:270
#, python-format
msgid ""
"The RRDTool process was terminated abnormally with exit code %(rcode)s and "
"message: %(msg)s"
msgstr ""

#: src/vigilo/connector_metro/rrdtool.py:277
msgid "Restarting..."
msgstr ""

#: src/vigilo/connector_metro/snmp.py:67
msgid "Connection to the SNMP daemon established"
msgstr ""

#: src/vigilo/connector_metro/snmp.py:135
#, python-format
msgid "Received OID outside my base: %s"
msgstr ""

#: src/vigilo/connector_metro/snmp.py:169
msgid "SNMP to RRDTool gateway started"
msgstr ""

#: src/vigilo/connector_metro/snmp.py:180
msgid "SNMP to RRDTool gateway stopped"
msgstr ""

#: src/vigilo/connector_metro/snmp.py:197
#, python-format
msgid "RRD directory does not exist: %s"
msgstr ""

#: src/vigilo/connector_metro/snmp.py:238
#, python-format
msgid "this does not look like an RRD file path: %s"
msgstr ""

#: src/vigilo/connector_metro/vigiconf_settings.py:48
msgid "No configuration database yet!"
msgstr ""

